packages/steiger-plugin-fsd/src/insignificant-slice/index.ts

```diff
@@ -93,3 +101,25 @@ async function traceSliceReferences(root: Folder) {
 
   return references
 }
+
+function resolveRelativePathsInConfigExtensions(tsconfig: TsConfigExcerpts, configPath: string) {
+  if (!tsconfig || !tsconfig.extends || !tsconfig.compilerOptions) {
+    return tsconfig
+  }
+
+  const paths = tsconfig.compilerOptions?.paths ?? {}
+
+  const newPaths = Object.fromEntries(
+    Object.entries(paths).map(([key, values]) => {
+      return [key, values.map((value: string) => join(configPath, '..', value))]
```

Comment:
issue: the `'..'` part looks hard-coded for the Nuxt case. I don't fully understand how this function works and how it fixes the issue